### PR TITLE
Added details on Ingress Operator and IPV4/6 modes

### PR DESCRIPTION
**Pull request by ahardin-rh**
Adding details per comments from @danehans in https://issues.redhat.com/browse/NE-248

### CODE REVIEW

File: modules/ingress-operator.adoc [hunk 1]
```diff
@@ -44,3 +44,19 @@ creates the deployment for the router:
...
+The Ingress Operator uses the `clusterNetwork` `cidr` from the cluster or
+network status to determine what mode (IPv4, IPv6, or dual stack) the managed
+Ingress controller, or router, should operate in. For example, if
+`clusterNetwork` contains only a v6 `cidr`, then the Ingress controller will
+operate in v6-only mode. In the following example, Ingress controllers managed
+by the Ingress Operator will run in v4-only mode because only one cluster
+network exists and the network is a v4 `cidr`:
+
+----
+$ oc get network/cluster -o jsonpath='
```
Comment (jboxman) [inline modules/ingress-operator.adoc:57]: This is probably:

```
oc get network/cluster -o jsonpath='{.status.clusterNetwork[*]}'
```

File: modules/ingress-operator.adoc [hunk 2]
```diff
@@ -44,3 +44,19 @@ creates the deployment for the router:
 ----
 $ oc get deployment -n openshift-ingress
 ----
+
+The Ingress Operator uses the `clusterNetwork` `cidr` from the cluster or
```
Comment (jboxman) [inline modules/ingress-operator.adoc:48]: And this is an array:

```
clusterNetwork[].cidr
```

**Comment by ahardin-rh** [comment]
@danehans PTAL Thanks!

**Comment by ahardin-rh** [comment]
@lihongan Can you please provide QE review? Thanks!

**Comment by lihongan** [comment]
LGTM. thank you @ahardin-rh

File: modules/ingress-operator.adoc [hunk 3]
```diff
@@ -44,3 +44,17 @@ creates the deployment for the router:
 ----
 $ oc get deployment -n openshift-ingress
 ----
+
+The Ingress Operator uses the `clusterNetwork[].cidr` from the cluster or
+network status to determine what mode (IPv4, IPv6, or dual stack) the managed
```
Comment (danehans) [inline modules/ingress-operator.adoc:49]: "the cluster or network status" > "network/cluster status"

File: modules/ingress-operator.adoc [hunk 4]
```diff
@@ -44,3 +44,17 @@ creates the deployment for the router:
...
+Ingress controller, or router, should operate in. For example, if
+`clusterNetwork` contains only a v6 `cidr`, then the Ingress controller will
+operate in v6-only mode. In the following example, Ingress controllers managed
+by the Ingress Operator will run in v4-only mode because only one cluster
```
Comment (danehans) [inline modules/ingress-operator.adoc:53]: "Ingress controllers" > "ingress controllers"

File: modules/ingress-operator.adoc [hunk 5]
```diff
@@ -44,3 +44,17 @@ creates the deployment for the router:
 ----
 $ oc get deployment -n openshift-ingress
 ----
+
+The Ingress Operator uses the `clusterNetwork[].cidr` from the cluster or
+network status to determine what mode (IPv4, IPv6, or dual stack) the managed
+Ingress controller, or router, should operate in. For example, if
+`clusterNetwork` contains only a v6 `cidr`, then the Ingress controller will
```
Comment (danehans) [inline modules/ingress-operator.adoc:51]: "Ingress controller, or router, should operate in" > "ingress controller, i.e. router, should operate in"

**Pull request merged**